sys/invitation.js

```diff
@@ -154,11 +163,10 @@ class FWInvitation {
           return {
             status: "expired",
           };
-        } else {
-          return {
-            status: "pending"
-          };
-        }
+        } 
+      }
+      return {
```

Comment:
need to continue supporting legacy license mode, since many early adopters are still using old license mode